task-1/app.py

```diff
@@ -0,0 +1,87 @@
+from flask import Flask, jsonify, request
+import json
+
+
+class InterfaceAPI(Flask):
+    def __init__(self, import_name):
+        super(InterfaceAPI, self).__init__(import_name)
+
+app = InterfaceAPI(__name__)
+
+def get_all_interfaces():
+    io = open("data.json","r")
+    string = io.read()
+    dictionary = json.loads(string)
+    io.close()
+    return dictionary['ietf-interfaces:interfaces']['interface']
```

Comment:
@tomowex use with open instead of reading data and closing it like this. also please use some more descriptive variable names
dictionary -> interfaces